plugin-server/src/utils/db/sql.ts

```diff
@@ -65,7 +65,9 @@ export async function getPluginRows(hub: Hub): Promise<Plugin[]> {
         WHERE posthog_plugin.id IN (${pluginConfigsInForceQuery('plugin_id')}
         GROUP BY posthog_pluginconfig.plugin_id)`,
         undefined,
-        'getPluginRows'
+        'getPluginRows',
+        undefined,
```

Comment:
noob ts question: can't we make the client be a key-value arg and not provide it?

Reply:
something maybe for a follow-up refactor - as it stands any change to `client` would require some refactoring of all the callsites